solutions/3727/01.py

```
@@ -0,0 +1,24 @@
+class Solution:
+    def maxAlternatingSum(self, nums: List[int]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)